.vib/harbor/cypress/cypress/integration/harbor_spec.js

```diff
@@ -0,0 +1,61 @@
+/// <reference types="cypress" />
+import {
+  random,
+} from './utils'
+
+it('allows user to log in and log out', () => {
+  cy.login();
+  cy.contains('div', 'Invalid user name or password.').should('not.exist');
+  cy.contains('button','admin').click();
+  cy.contains('a', 'Log Out').click();
+  cy.get('.login-group');
```

Comment:
What does this do after getting?

Reply:
I'm using this as a check to verify the logout was correct and it redirected you to the login page. I updated the element for it to be clearer.